interfaces/java/CMakeLists.txt

```diff
@@ -168,8 +168,11 @@ if(HELICS_ENABLE_SWIG AND SWIG_EXECUTABLE)
 
     if(MSVC)
         target_compile_options(${SWIG_MODULE_helicsJava_REAL_NAME} PRIVATE "/wd4100")
-    elseif(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
-        target_compile_options(${SWIG_MODULE_helicsJava_REAL_NAME} PRIVATE -Wno-long-long -Wno-strict-aliasing -Wno-deprecated-declarations)
+     else()
+	    target_compile_options(${SWIG_MODULE_helicsJava_REAL_NAME} PRIVATE -Wno-deprecated-declarations)
+		if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
+            target_compile_options(${SWIG_MODULE_helicsJava_REAL_NAME} PRIVATE -Wno-long-long -Wno-strict-aliasing)
+		endif()
     endif()
```

Comment:
```suggestion
     else()
          target_compile_options(${SWIG_MODULE_helicsJava_REAL_NAME} PRIVATE -Wno-deprecated-declarations)
          if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
               target_compile_options(${SWIG_MODULE_helicsJava_REAL_NAME} PRIVATE -Wno-long-long -Wno-strict-aliasing)
          endif()
    endif()
```

Reply:
This is where having the automatic cmake-format would be nice